package.json

```diff
@@ -31,6 +33,7 @@
     "truffle-contract": "^4.0.29"
   },
   "dependencies": {
+    "@gnosis.pm/safe-contracts": "^1.0.0",
```

Comment:
Let's avoid unused packages. I know you want to use this in the future, but then I'd rather clean up here and take it back when the time comes one day 👍